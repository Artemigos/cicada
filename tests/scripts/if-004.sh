# NOTE: this script will print two lines into stderr,
# which will not break the tests
x=foo
if echo hi
    if [ $x = foo ]
        echo before printing continue
        continue
        echo after printing continue
    fi
    echo end of outer if
fi
echo bye

# just a comment
continue

echo bye bye
