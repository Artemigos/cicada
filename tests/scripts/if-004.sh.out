hi
before printing continue
after printing continue
end of outer if
bye
bye bye
